### PR TITLE
Control Changes

### PR DESCRIPTION
**Pull request by Strflightmight09**

### CODE REVIEW

File: src/main/java/frc/trigon/robot/commands/factories/ShootingCommands.java [hunk 1]
```diff
@@ -81,6 +81,17 @@ private static Command getFeedNoteForCloseSpeakerShotCommand() {
         return GeneralCommands.runWhen(IntakeCommands.getSetTargetStateCommand(IntakeConstants.IntakeState.FEED_SHOOTING).alongWith(getVisualizeNoteShootingCommand()), () -> RobotContainer.SHOOTER.atTargetVelocity() && RobotContainer.PITCHER.atTargetPitch());
     }
 
+    public static Command getShootManualLowDeliveryCommand() {
+        return new ParallelCommandGroup(
```
Comment (levyishai) [inline src/main/java/frc/trigon/robot/commands/factories/ShootingCommands.java:78]: Maybe put this above since it's public?

**Review by levyishai — CHANGES_REQUESTED** [review]

File: src/main/java/frc/trigon/robot/commands/factories/ShootingCommands.java (hunk 1 above)
Comment (Strflightmight09) [reply src/main/java/frc/trigon/robot/commands/factories/ShootingCommands.java:78]: I have it organized by shooting type. The normal shooting is all on the top, then close shot stuff, and then manual delivery.

Comment (levyishai) [reply src/main/java/frc/trigon/robot/commands/factories/ShootingCommands.java:78]: I didn't notice that at first, and missed some commands. 
1. You should make the prepare command private
2. Maybe a more readble way would be to put the public commands at the top, and then similar commands next to each other. Right now, I've missed some public commands.

File: src/main/java/frc/trigon/robot/commands/factories/ShootingCommands.java [hunk 2]
```diff
@@ -32,7 +32,29 @@ public static Command getShootAtShootingTargetCommand(boolean isDelivery) {
...
+    }
+
+    public static Command getCloseSpeakerShotCommand() {
+        return new ParallelCommandGroup(
+                getPrepareCloseSpeakerShotCommand(),
+                getFeedNoteForCloseSpeakerShotCommand()
+        );
+    }
+
+    public static Command getShootManualLowDeliveryCommand() {
```
Comment (levyishai) [inline src/main/java/frc/trigon/robot/commands/factories/ShootingCommands.java]: getManualLowDeliveryCommand

**Review by levyishai — CHANGES_REQUESTED** [review]
Put similar functions next to each other. For example, first putting the prepare for shooting, then prepare for close shot, then for delivery. And then, feed note for shooting, feed note for delivery, etc.

File: src/main/java/frc/trigon/robot/commands/factories/ShootingCommands.java [hunk 3]
```diff
@@ -32,7 +32,29 @@ public static Command getShootAtShootingTargetCommand(boolean isDelivery) {
...
+        return new ParallelCommandGroup(
+                ShooterCommands.getSetTargetVelocityCommand(ShootingConstants.MANUAL_LOW_DELIVERY_SHOOTING_ROTATIONS_PER_SECOND),
```
Comment (levyishai) [inline src/main/java/frc/trigon/robot/commands/factories/ShootingCommands.java]: You would also want a set pitcher angle.

**Review by levyishai — CHANGES_REQUESTED** [review]

File: src/main/java/frc/trigon/robot/commands/factories/ShootingCommands.java [hunk 4]
```diff
@@ -32,7 +32,30 @@ public static Command getShootAtShootingTargetCommand(boolean isDelivery) {
...
+        return new ParallelCommandGroup(
+                getPrepareCloseSpeakerShotCommand(),
+                getFeedNoteForCloseSpeakerShotCommand()
+        );
+    }
+
+    public static Command getManualLowDeliveryCommand() {
+        return new ParallelCommandGroup(
+                ShooterCommands.getSetTargetVelocityCommand(ShootingConstants.MANUAL_LOW_DELIVERY_SHOOTING_ROTATIONS_PER_SECOND),
+                PitcherCommands.getSetTargetPitchCommand(ShootingConstants.MANUAL_LOW_DELIVERY_PITCH),
```
Comment (levyishai) [inline src/main/java/frc/trigon/robot/commands/factories/ShootingCommands.java]: Add a prepare command for this.

**Review by levyishai — CHANGES_REQUESTED** [review]

**Review by levyishai — APPROVED** [review]

**Review by levyishai — APPROVED** [review]

**Review by Nummun14 — APPROVED** [review]

**Pull request merged**